components/scream/cime_config/buildnml

```diff
@@ -505,6 +505,13 @@ def create_raw_xml_file(case, caseroot):
         with open(raw_xml_file, "w") as fd:
             ET.ElementTree(raw_xml).write(fd, method='xml', encoding="unicode")
 
+    # Now that we have our namelist_scream.xml file, we can apply buffered
+    # atmchange requests.
+    atmchg_buffer = case.get_value("SCREAM_ATMCHANGE_BUFFER")
+    if atmchg_buffer:
+        run_cmd_no_fail("{}/atmchange {}".format(caseroot, atmchg_buffer))
```

Comment:
Does this assume that there's only one atmchange cmd stored in the xml temp var?

Reply:
I.e., does it work correctly with 
```
./xmlchange SCREAM_ATMCHANGE_BUFFER='cubed_sphere_map=42'
./xmlchange --append SCREAM_ATMCHANGE_BUFFER='cubed_sphere_map=24'
```
?

Reply:
@bartgol , I just tested it and it appears to work. xmlchange is even smart enough to add a whitespace when appending things:

```
% ./xmlquery SCREAM_ATMCHANGE_BUFFER
	SCREAM_ATMCHANGE_BUFFER: 
% ./xmlchange SCREAM_ATMCHANGE_BUFFER='foo=bar'
% ./xmlquery SCREAM_ATMCHANGE_BUFFER
	SCREAM_ATMCHANGE_BUFFER: foo=bar
% ./xmlchange SCREAM_ATMCHANGE_BUFFER='baz=bat' --append
% ./xmlquery SCREAM_ATMCHANGE_BUFFER
	SCREAM_ATMCHANGE_BUFFER: foo=bar baz=bat
```

Reply:
Oh, I see you wanted me to test case.setup, not xmlchange. That appears to work too:

```
% ./xmlchange SCREAM_ATMCHANGE_BUFFER='cubed_sphere_map=32'
% ./xmlchange SCREAM_ATMCHANGE_BUFFER='dt_remap_factor=42' --append
% ./xmlquery SCREAM_ATMCHANGE_BUFFER
	SCREAM_ATMCHANGE_BUFFER: cubed_sphere_map=32 dt_remap_factor=42
% /bin/rm namelist_scream.xml 
% ./case.setup
% grep -E '>(42|32)' namelist_scream.xml
    <cubed_sphere_map type="integer">32</cubed_sphere_map>
    <dt_remap_factor constraints="ge 1" type="integer">42</dt_remap_factor>
```